crates/libcgroups/src/v2/cpu.rs

```diff
@@ -96,6 +97,10 @@ impl Cpu {
             common::write_cgroup_file(path.join(CGROUP_CPU_BURST), burst)?;
         }
 
+        if let Some(idle) = cpu.idle() {
+            common::write_cgroup_file(path.join(CGROUP_CPU_IDLE), idle)?;
```

Comment:
Same here. We should add or adapt the test cases to account for the new configuration.